src/cli/commands/pack.js

```diff
@@ -93,13 +100,25 @@ export async function packTarball(
   }
 
   const files = await fs.walk(config.cwd, null, new Set(FOLDERS_IGNORE));
+
+  if (globalGitIgnoreFile) {
+    const stat = await lstat(globalGitIgnoreFile);
+    const loc = path.resolve(globalGitIgnoreFile);
+    const basename = path.basename(globalGitIgnoreFile);
+    files.push({
```

Comment:
Does this put the global gitignore file in the packed archive? Is there a reason to include this? When it's unpacked in someone's `node_modules` directory they wouldn't be using it as a git archive, so it doesn't seem like we would need to persist this.